File: src/string/countDigits/TestCountDigits.java
package string.countDigits;

import static org.junit.Assert.assertEquals;
import static string.countDigits.CountDigits.countDigits;

import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.CsvSource;

public class TestCountDigits {
    @ParameterizedTest
    @CsvSource(value = {
            "nezabud dl-6, 1",
            "l33t 3301, 6",
            "1029384756aaa1234567890, 20",
            "jehfkjshfkjsdk90909090, 8"
    })
    void testCountDigits(String input, int expected) {
        assertEquals(expected, countDigits(input));
    }
}

File: src/string/countDigits/test_count_digits.py
from count_digits import *
import pytest


@pytest.mark.parametrize("input, expected", [
                         ('nezabud dl-6', 1),
                         ('l33t 3301', 6),
                         ('1029384756aaa1234567890', 20),
                         ('jehfkjshfkjsdk90909090', 8)
                         ])
def test_count_digits(input, expected):
    assert count_digits(input) == expected
